_data/domains.yml

```diff
@@ -63,6 +63,15 @@ websites:
       twitter: CrazyDomains
       facebook: crazydomains
 
+    - name: CSC
+      url: https://www.cscglobal.com/cscglobal/home/
+      img: csc-logo.png
+      tfa: Yes
```

Comment:
Could you provide any documentation page for how to enable 2FA?

Reply:
Unfortunately, we do not publish any documentation related to 2FA. The instructions are provided to our customers during onboarding.

Reply:
In that case I'll need some sort of screenshot of the setup process or login process (with any personal information blurred out).
You can post it here as a comment by dragging and dropping the image to the message field.

Reply:
I don't see anything related to `hardware` here. Also, should we have an exception for requiring the Duo app?

Reply:
@brycx no the duo app is still software 😛

Reply:
Hardware is accomplished through key fobs provided by Duo on request. Very few of our customers use hardware key fobs because there is an extra cost.

If a customer has a key fob, they can enter the passcode provided by the key fob via the third option in the image above.

The Duo app is only required for the "Duo Push" option. Neither phone call, SMS, or hardware require the user have the Duo app. That being said, I'll let you guys decide if this should be considered an exception.

Reply:
Given what you've said, the current state sounds reasonable.